frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetDetails.test.tsx

```diff
@@ -69,6 +69,49 @@ const clusterSetCluster: ManagedCluster = mockManagedClusters.find(
     (mc: ManagedCluster) => mc.metadata.labels?.[managedClusterSetLabel] === mockManagedClusterSet.metadata.name!
 )!
 
+const mockManagedClusterRosa: ManagedCluster = {
```

Comment:
can we add a UT for ARO cloud too? Just to ensure future changes don't break anything related to ARO.

Reply:
Yeah, good idea.
Updated the PR